.scripts/process_build.sh

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env bash
+
+# Sorry, everyone
+if (( ${BASH_VERSION%%.*} < 4 )); then
+  echo "This script requires Bash >= 4. On OSX, try: brew install bash"
+  exit 1
+fi
+
+# Any error is fatal.
+set -e
+set -o pipefail
+
+BUILD_DIR=$(pwd)
+if [[ -f ~/.bash_profile ]]; then
+  source ~/.bash_profile
+fi
+cd "$BUILD_DIR"
+
+# Constants for showing color in output window
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+CURRENT_VERSION=$(head -n 1 version.txt)
+
+echo "*** Clean up log files ***"
+if [[ -f src/ocaml-output/fstar/MicroBenchMarkOutput.log ]]; then
```

Comment:
Super minor comment: `rm -f` returns a zero exit code even if the file doesn't exist, so you can replace

```
if [[ -f foo ]]; then rm foo; fi
```

with

```
rm -f foo
```

Up to you, no strong feelings one way or another